src/Generics/Linear/TH.hs

```diff
@@ -569,5 +570,6 @@ Why do we do this? Imagine what the instance would be without the explicit retur
   instance Generic1 S where
     type Rep1 S = ... (Rec0 k)
 
-This is an error, since the variable k is now out-of-scope!
+This is an error, since the variable k is now out-of-scope! The TypeInTypeSpec
+test case checks that this behaves as intended.
```

Comment:
Do we need the signature on the instance head, or just the type family argument?

Reply:
In general we need both, although this particular example only needs the kind annotation on the type family instance. I've modified the wording to be a bit more precise here.